docs/howto/manage-channels.rst

```diff
@@ -79,14 +79,14 @@ You can request a track guardrail and create a track.
 Open a channel
 --------------
 
-A channel is opened implicitly when you release a revision to it.
+A channel is opened when you release a revision to that channel. Before that, the
+channel is created but not opened.
 
 
 Close a channel
 ---------------
 
-A channel is opened when you release a revision to that channel. Before that, the
-channel is created but not opened. When you're closing a channel, e.g.,
+When you're closing a channel, e.g.,
```

Comment:
I need help here. That is a How To. But we do not explain how to close the track. If you suggest I can rephrase it